pipelines/nextflow/modules/query_core_meta.nf

```diff
@@ -0,0 +1,34 @@
+/*
+See the NOTICE file distributed with this work for additional information
```

Comment:
general comment on this module: all the metakeys were used before to build the folder structure to make our life easier in order to sync the results into the FTP. As this is no longer needed we can avoid this step and the only thing we can use is the core db or the GCA that can be retrieved from the db name. So we can avoid calling genomio and anyway I think this  task can be moved to a simply python script if we still want to retrieve the metakeys
So I would skip the module or add a python script for this in the bin folder

Reply:
Not every core database has the right accession in their name (e.g. GCF are not allowed in main sites), so I'm afraid the solution you are proposing does not work for all our core databases at the minute (once main sites are frozen and we are only importing new things for Beta that will be different).

If you want to make the pipeline completely disconnected from the core dbs, we could decide to use something like datasets CLI to fetch the assembly FASTA file from GenBank giving just the accession and scientific name in the input file. But I would argue that is out of the scope of this PR.